### ISSUE TITLE
Dynamic Schema 

### ISSUE THREAD

**Issue by cmandesign** [graphql-python/graphene#742]
I almost spent 3 days to find a way for creating a dynamic schema in python graphene. the only related result I could find is the below link: https://github.com/graphql-python/graphene/blob/master/graphene/types/dynamic.py But I couldn't find any documentation for it.

The whole idea is to create a dynamic schema. I want to provide a GraphQL compatible API that makes users able to query my contents even if Models are not defined in the code. In other words, I want to create Models on the fly. I have no idea about what shall I do.

It would be a great favor if you can provide an example for that.

Update :

My Project is a Headless CMS which has a feature that users can create their own content types and I want to provide a GraphQL interface to make everything easier and more flexible.

Here is example of my Content Types in DB :
```
{
  "id": "author",
  "name": "Book Author",
  "desc": "",
  "options":[
    {
      "id": "author_faname",
      "label": "Sample Sample",
      "type": "text",
      "required": true,
      "placeholder":"One Two Three Four"
    },
    {
      "id": "author_enname",
      "label": "Sample label",
      "type": "text",
      "required": true,
      "placeholder":"Sample Placeholder"
    }
  ]
}
```

And Here is Stored content in DB based on that content type :

```
{
  "id": "9rqgbrox10",
  "content_type": "author",
  "data":{
    "author_fname":"Jimmy",
    "author_ename":"Hello"
  }
}
```

Now as my Models are not declared in Code and they are completely in DB, I want to make my schemas on the fly and I don't know what is best the solution for this. I know there should be a way because the other Headless CMS Projects are providing this.

Thanks in advance!

**Comment by zflamig** [comment]
We are doing what you describe, our code might be a little bit hard to follow but you can start from https://github.com/uc-cdis/peregrine/blob/master/peregrine/resources/submission/graphql/node.py#L705 hopefully it makes sense...

**Comment by enf644** [comment]
Here is answer on stackoverflow - https://stackoverflow.com/questions/50559580/creating-dynamic-schema-on-runtime-graphene/52690104#52690104
